numpy/lib/npyio.py

```diff
@@ -1161,10 +1161,10 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
         while such lines are counted in `skiprows`.
 
         .. versionadded:: 1.16.0
-        
+
         .. versionchanged:: 1.23.0
-            Lines containing no data, including comment lines (e.g., lines 
-            starting with '#' or as specified via `comments`) are not counted 
+            Lines containing no data, including comment lines (e.g., lines
+            starting with '#' or as specified via `comments`) are not counted
             towards `max_rows`.
     quotechar : unicode character or None, optional
         The character used to denote the start and end of a quoted item.
```

Comment:
While I agree with the whitespace cleanup in files you are changing anyway, I don't think this one should be part of this PR.

Reply:
Oh, not even an actual change left in the end here...  (`set_module` is imported via `overrides` here, which seemed OKish.)